{"uuid":"4e6a3ada-b22d-46c5-ac3f-13ad996d5777","historyId":"e6d498b70e1d2a227422ca2162c5a38e","fullName":"com.leadAngel.testCases.TC_SANITY_LoginTest.SanityloginTest","labels":[{"name":"package","value":"com.leadAngel.testCases.TC_SANITY_LoginTest"},{"name":"testClass","value":"com.leadAngel.testCases.TC_SANITY_LoginTest"},{"name":"testMethod","value":"SanityloginTest"},{"name":"parentSuite","value":"LeadAngelV1"},{"name":"suite","value":"LA_Chrome"},{"name":"subSuite","value":"com.leadAngel.testCases.TC_SANITY_LoginTest"},{"name":"host","value":"DESKTOP-SV17MOP"},{"name":"thread","value":"15736@DESKTOP-SV17MOP.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"},{"name":"feature","value":"LoginTest SANITY"},{"name":"epic","value":"Sanity"}],"links":[],"name":"All Positive Cases of SSO, Forgot Password & Login Section","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"The following asserts failed:\n\texpected [true] but found [false]","trace":"java.lang.AssertionError: The following asserts failed:\n\texpected [true] but found [false]\r\n\tat org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)\r\n\tat org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)\r\n\tat com.leadAngel.testCases.TC_SANITY_LoginTest.SanityloginTest(TC_SANITY_LoginTest.java:71)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)\r\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:568)\r\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)\r\n\tat org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)\r\n\tat org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)\r\n\tat org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)\r\n\tat org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)\r\n\tat org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)\r\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)\r\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1511)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:766)\r\n\tat org.testng.TestRunner.run(TestRunner.java:587)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:384)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:286)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1109)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1039)\r\n\tat org.testng.TestNG.run(TestNG.java:1007)\r\n\tat org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)\r\n\tat org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)\r\n\tat org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)\r\n"},"stage":"finished","description":"Login Module Sanity Test Description","steps":[],"attachments":[{"name":"SanityloginTest","source":"5c5fbfeb-9ec9-40d5-b09c-d3d4c6987712-attachment"},{"name":"SanityloginTest","source":"eba52dd4-7b00-4f04-8dbb-98deb7097793-attachment"}],"parameters":[{"name":"roleName","value":"admin"},{"name":"browser","value":"chrome"}],"start":1698905586770,"stop":1698905689227}